sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -52,6 +52,8 @@ import {
   OFFERING_STUDY_PERIOD_MAX_DAYS,
   OFFERING_STUDY_PERIOD_MIN_DAYS_FULL_TIME,
   OFFERING_STUDY_PERIOD_MIN_DAYS_PART_TIME,
```

Comment:
Also the validator `studyPeriodMinLength` is not required anymore and it needs to be removed unless I am missing something. 

![image](https://github.com/user-attachments/assets/be4dc835-6c4b-4912-b68f-b8a0fad0408c)